### DISCUSSION TITLE
SwiftUI 강의 투표

### DISCUSSION THREAD

**Discussion by EunsuSeo01** [A-Yo-Dan/SwiftUI_Study#16] [Polls]
현재 선택한 강의: https://www.udemy.com/course/swiftui-masterclass-course-ios-development-with-swift/

램프(박민서)의 의견 - 이 강의는 뭔가 라이브 코딩 듣는 느낌..? 진행하면서 가끔씩 이 코드가 뭔지 설명해주긴 하는데 완벽한 설명이 아니다.
'SwiftUI로 이렇게 만드는구나'의 느낌이라 각각 컴포넌트들이 어떤 식으로 작동하는지 어떤 걸 활용하는지에 대해선 완벽히 얘기를 안 해준다.
이 강의로 진행하려면 각 컴포넌트들에 대해서는 따로 공부를 하고 프로젝트 클론코딩을 해야할 거 같다.
SwiftUI를 공부하는 거면 전에 얘기했던 강의(SwiftUI - Declarative Interfaces for any Apple Device)가 더 나을 것 같기도?
각자 섹션 6,7 보면서 생각 얘기해 줄 수 있는지!

Discussions 테스트 겸 추가해보다 킥킥

**Comment by FpRaArNkK** [comment]
갠적으로는 개념공부 진행하면서 사이드로 저걸하면 되게 도움되겠다라는 생각임다..!
개념 강의 열심히 듣고 프로젝트 들어보면서 스유로 이런식으로 만드는구나 참고하는?

**Reply by FpRaArNkK** [reply]
SwiftUI - Declarative Interfaces for any Apple Device 강의 링크입니다!

https://www.udemy.com/course/swiftui-declarative-interfaces-for-any-apple-device/

**Reply by youz2me** [reply]
그럼 강의 두개를 듣는건가 ??? 메인은 Declarative로 가고 Masterclass를 서브로 ...................
다 좋은데 시간이 될지 모르겠군 ㅜㅜㅜ

**Comment by youz2me** [comment]
방금 강의를 좀 보고 왔는데 왜 라이브 코딩이라고 했는지 알 것 같네요 ^__^...
후기에도 자칫하면 따라만 하고 끝나는 강의라고 한걸 본 것 같은데 이래서였구나
개념 공부 진행하면서 사이드로 하는거 좋은 것 같아 !!!! 근데 그러면 1주차 진행해보고 진도 조정이 불가피하긴 할 것 같군

**Comment by youz2me** [comment]
완벽한 우리 스터디 짱

**Comment by hamgui-2022** [comment]
가격의 부담이 조금 있는 듯 합니다..

**Comment by FpRaArNkK** [comment]
땅땅땅 기존 강의로 진행하는 걸로 가자~
